.vscode/extensions.json

```diff
@@ -0,0 +1,17 @@
+{
+	"recommendations": [
```

Comment:
I don't think this is necessary for our project at our scale. I'd rather leave the extensions up to everyone else's preferences.

Reply:
It doesn't force you to install them, it just gives a popup that people new to vscode can use to easily install all the extensions

Reply:
I'm aware, but those pop-ups are annoying tbh. Also, some of these extensions won't even be able to be used by the common programmer on their school laptop, such as GitLens. I don't want to confuse them